.github/actions/spelling/expect/expect.txt

```diff
@@ -2287,6 +2287,7 @@ YCENTER
 YCount
 YDPI
 YOffset
+ype
```

Comment:
I'm surprised by this -- we already have `Type` entries in the natvis. Maybe @jsoref knows?

Reply:
It's from:
```
    <ClInclude Include="..\..\inc\til\type_traits.h" />
```

You can add `\\type` to patterns for now.

The next version of check-spelling won't default to treating `\n`, `\r`, and `\t` as if they're c++ escapes.

Reply:
(The next version will also do a better job of explaining where these [substrings](https://github.com/check-spelling/check-spelling/wiki/Feature%3A-Warning-about-substring-matches) are coming from in case you do choose to retain that [pattern](https://github.com/check-spelling/spell-check-this/blob/92f1181eb1225c22c08d71e61d6468fec100484a/.github/actions/spelling/candidate.patterns#L536-L538)...)

Reply:
_oh, interesting_. Broadly applying escaping rules to some languages that don't follow the same escaping rules! A mistake I've made too many times :smile:

Thanks for the info, as always

Reply:
Yeah. For the next version, I'm throwing in the towel, you'll have to opt into that rule instead.

Handling rules per file type is something I don't think I'll actively support (you can do it w/ a matrix, I have one or two projects that demo it). I could _probably_ do it w/ a magical comment header, but ...

Reply:
Anyway, that's what this file is working around, so this is where you'd add the entry: https://github.com/microsoft/terminal/blob/dev/lhecker/til-generational/.github/actions/spelling/patterns/0_t.txt